tests/input_converter/test_converter.py

```diff
@@ -582,3 +491,136 @@ def test_convert_wind_to_component_zero_values(self, local_study_w_areas, fr_win
         wind_components, _ = converter._convert_wind_to_component_list(areas)
 
         assert wind_components == []
+
+    def test_convert_links_to_component(self, local_study_w_links):
+        _, converter = self._init_area_reading(local_study_w_links)
+        study_path = converter.study_path
+        (
+            links_components,
+            links_connections,
+        ) = converter._convert_link_to_component_list()
+
+        fr_it_direct_links_timeseries = str(
+            study_path / "input" / "links" / "fr" / "capacities" / "it_direct.txt"
```

Comment:
You can introduce a constant for `study_path / "input" / "links" / "fr" / "capacities" /`